src/components/Charts/AreaChart/AreaChart.js

```diff
@@ -222,16 +222,22 @@ export default class AreaChart extends React.Component {
 }
 
 AreaChart.propTypes = {
+  /** Height of the Chart in px. Accepts only numbers. */
+  height: PropTypes.number,
+  /** Width of the Chart in px. Accepts only numbers. */ 
+  width: PropTypes.number,
   /** Turns, on/off animation and allows for selection of different types of animations. */
   animation: PropTypes.oneOf([false, 'noWobble', 'gentle', 'wobbly', 'stiff']),
-  /** Color to be used on all chart lines, unless colorRange is provided */
+  /** Color to be used on all chart lines, unless colorRange is provided 
+   * @uxpinignoreprop 
+  */
```

Comment:
1. when multiline comment is made it's good to leave first lint empty like:
```
/**
 * Description here
 */
```

2. Missing space on line 233 (`*` should be aligned)